README.md

```diff
@@ -4,39 +4,58 @@ Hartsfield humbly supports UC Berkeley's DataHub.
 
 ![Hartsfield, re-imagined as a field of hearts.](src/assets/hEartsfield.png)
 
-## Installation
 
-* Install Python 3.8
-* Create your virtual environment (venv)
-* Install dependencies
+
+
+## To run this locally:
+
+### Install a python 3.11 venv in the project directory, activate it, and install requirements:
 
 ```
-pip3 install -r requirements.txt [--upgrade]
+python3.11 -m venv ./venv
+source ./venv/bin/activate
+pip install -r requirements.txt
 ```
 
+### Install npm (the Node.js package manager), adjust the version, install the project dependencies, and do the "audit fix" if it complains that you should do so:
 
+https://docs.npmjs.com/downloading-and-installing-node-js-and-npm
 
-### Create local configurations
+```
+npm install -g npm@9.8.1
+npm install
+npm audit fix
+```
+
+### Securely install local configurations with secrets
 
-If you plan to use any resources outside localhost, put your configurations in a separately encrypted area:
+Put your configurations in a separately encrypted area outside of the project folder, which you will later export to environment variables. Ensure that your uid is in the AUTHORIZED_USERS list within that file.
 
 ```
 mkdir /Volumes/XYZ/hartsfield_config
-export HARTSFIELD_LOCAL_CONFIGS=/Volumes/XYZ/hartsfield_config
 ```
 
-## Greg does a jam like this from a pair of terminals in VSCode to run this locally:
+### Run one terminal session for the python back end...
 
 ```
 source venv/bin/activate
-export HARTSFIELD_LOCAL_CONFIGS=/Users/gregm/rip_hartsfield/hartsfield_config
+source .env.development
+export HARTSFIELD_LOCAL_CONFIGS=/Volumes/XYZ/hartsfield_config
 export HARTSFIELD_ENV=development
 venv/bin/python application.py
 ```
-and
+### ...and another terminal session for the Node.js front end:
 ```
-source venv/bin/activate
-export HARTSFIELD_LOCAL_CONFIGS=/Users/gregm/rip_hartsfield/hartsfield_config
+source .env.development
+export HARTSFIELD_LOCAL_CONFIGS=/Volumes/XYZ/hartsfield_config
 export HARTSFIELD_ENV=development
 npm run serve-vue
 ```
```

Comment:
`source .env.development` seems unnecessary in both instances above. Vite should pick it up automatically when `npm run serve-vue`